### PR TITLE
Feature/exit keys

### PR DESCRIPTION
**Pull request by Rephobia**
Hello, thanks for this tool. I use it in my expose script and missprints are annoying. This pull request adds the `--exit-keys` option. The option takes a space separated sequences where each sequence separated by '+', eg `--exit-keys Control_L+g Shift+f`.  Keys pressing don't equal to the option sequences will not cause the tool to exit. If the option isn't passed, behavior is old.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -255,27 +256,21 @@ fn main() {
...
-                        let kstr = unsafe {
-                            CStr::from_ptr(x11::xlib::XKeysymToString(ksym.into()))
-                                .to_str()
-                                .expect("Couldn't create Rust string from C string")
-                        };
-                        if ksym == xkb::KEY_Escape {
-                            closed = true;
-                        }
+                        let ksym = utils::get_pressed_symbol(&conn, &event);
+                        let kstr = utils::convert_to_string(ksym);
```
Comment (svenstaro) [inline src/main.rs:266]: I like how you snuck in this little cleanup.

**Review by svenstaro — CHANGES_REQUESTED** [review]
This is great! I only have small nits apart from the code lines: Could you add back the log lines you removed? I find them quite helpful when debugging the flow of the tool. Aside from that, I'm happy to merge as is. :)

**Comment by Rephobia** [comment]
Thanks for the review, i finalized the pull request

**Comment by svenstaro** [comment]
Good stuff, thanks!

**Pull request merged**